framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1504,6 +1516,41 @@ private Object convert(Number number, TypeKind expectedKind) {
         }
     }
 
+    /**
+     * Converts {@code number} to {@code expectedKind} * -1 if {@code negate}. Converts {@code
+     * number} to {@code expectedKind} if not {@code negate}.
+     *
+     * @param number Number value to be converted
+     * @param expectedKind converts the Number to {byte, short, int, long, char, float, double}
+     * @param negate converted value
```

Comment:
It seems odd to add a parameter, if the method is only ever invoked with `true`.
How about simply naming the method `convertNegated` and dropping the parameter?